### PR TITLE
feat: add gas averages over time into grafana

### LINKED ISSUE #12179: Include gas/s averages in execution throughput graph
Right now the execution throughput metric just records a raw gas metric, when it can have a good amount of variation. We should include averages over different time intervals (1m, 5m, 10m, ...) in the graph.

### PR DESCRIPTION
**Pull request by edisontim**
Closes #12179
Screenshot 2024-10-31 at 15 45 44

### CODE REVIEW

File: etc/grafana/dashboards/overview.json [hunk 1]
```diff
@@ -3016,6 +3016,86 @@
...
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+            },
+            "disableTextWrap": false,
+            "editorMode": "builder",
+            "expr": "avg_over_time(reth_sync_execution_gas_per_second{instance=~\"$instance\"}[1m])",
+            "fullMetaSearch": false,
+            "hide": false,
+            "includeNullMetadata": true,
+            "legendFormat": "avg/m",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:3031]:
```suggestion
            "legendFormat": "Avg Gas/s (1m)",
```

File: etc/grafana/dashboards/overview.json [hunk 2]
```diff
@@ -3016,6 +3016,86 @@
...
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+            },
+            "disableTextWrap": false,
+            "editorMode": "builder",
+            "expr": "avg_over_time(reth_sync_execution_gas_per_second{instance=~\"$instance\"}[5m])",
+            "fullMetaSearch": false,
+            "hide": false,
+            "includeNullMetadata": true,
+            "legendFormat": "avg/5m",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:3047]:
```suggestion
            "legendFormat": "Avg Gas/s (5m)",
```

File: etc/grafana/dashboards/overview.json [hunk 3]
```diff
@@ -3016,6 +3016,86 @@
...
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+            },
+            "disableTextWrap": false,
+            "editorMode": "builder",
+            "expr": "avg_over_time(reth_sync_execution_gas_per_second{instance=~\"$instance\"}[10m])",
+            "fullMetaSearch": false,
+            "hide": false,
+            "includeNullMetadata": true,
+            "legendFormat": "avg/10m",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:3063]:
```suggestion
            "legendFormat": "Avg Gas/s (10m)",
```

File: etc/grafana/dashboards/overview.json [hunk 4]
```diff
@@ -3016,6 +3016,86 @@
...
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+            },
+            "disableTextWrap": false,
+            "editorMode": "builder",
+            "expr": "avg_over_time(reth_sync_execution_gas_per_second{instance=~\"$instance\"}[1h])",
+            "fullMetaSearch": false,
+            "hide": false,
+            "includeNullMetadata": true,
+            "legendFormat": "avg/h",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:3079]:
```suggestion
            "legendFormat": "Avg Gas/s (1h)",
```

File: etc/grafana/dashboards/overview.json [hunk 5]
```diff
@@ -3016,6 +3016,86 @@
...
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+            },
+            "disableTextWrap": false,
+            "editorMode": "builder",
+            "expr": "avg_over_time(reth_sync_execution_gas_per_second{instance=~\"$instance\"}[24h])",
+            "fullMetaSearch": false,
+            "hide": false,
+            "includeNullMetadata": true,
+            "legendFormat": "avg/24h",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:3095]:
```suggestion
            "legendFormat": "Avg Gas/s (24h)",
```

**Review by Rjected — CHANGES_REQUESTED** [review]
thanks! just notes on the legend, also can we also add a 30m one?

**Review by Rjected — APPROVED** [review]
Awesome thanks!

**Review by mattsse — APPROVED** [review]

**Pull request merged**